README.md

```diff
@@ -1,2 +1,14 @@
 # gtfs-realtime-capsule
 Tool to archive GTFS-rt data
+
+# Local development environment setup via docker
```

Comment:
Could you also add a section on the process for installing new packages, and committing them to the repository? Since this is a docker container, is it going to need anything besides pip (like conda, pipenv) for package management?

Reply:
Add this as a follow-up. i'm merging this now so I can use it for my CR